src/javascript/JContent/JContent.accordion-items.jsx

```diff
@@ -80,10 +80,11 @@ export const jContentAccordionItems = registry => {
         label: 'jcontent:label.contentManager.navigation.pages',
         defaultPath: siteKey => '/sites/' + siteKey,
         getPathForItem: node => {
-            const pages = node.ancestors.filter(n => n.primaryNodeType.name === 'jnt:page');
+            const pages = node.ancestors
+                .filter(n => n.primaryNodeType.name === 'jnt:page');
             return pages[pages.length - 1].path;
         },
-        canDisplayItem: node => !filesRegex.test(node.path) && !contentsRegex.test(node.path),
+        canDisplayItem: ({selectionNode, folderNode}) => selectionNode ? !filesRegex.test(selectionNode.path) && !contentsRegex.test(selectionNode.path) : /^\/sites\/.*/.test(folderNode.path),
```

Comment:
still in the case of folderNode , this one should not match the ones that are matched by contentFolders and media - like /sites/xx/files (so not match the 2 regex from content-folder and media ?)